components/region_cache_memory_engine/src/engine.rs

```diff
@@ -61,6 +61,11 @@ impl RegionMemoryEngine {
                     arena_size,
                     true,
                 )),
+                Arc::new(Skiplist::with_capacity(
```

Comment:
Let's remove this due to the reason above.

Reply:
See above.